core/services/chainlink/application.go

```diff
@@ -161,6 +162,7 @@ func (app *ChainlinkApplication) Start() error {
 	// XXX: Change to exit on first encountered error.
 	return multierr.Combine(
 		app.Store.Start(),
+		app.Store.EthClient.Dial(context.TODO()),
```

Comment:
From the docs: `Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).` That's most of our codebase. We should probably default to `context.TODO()` instead of `context.Background()`, but I don't feel too strongly about it!

Reply:
OK I didnt know that!